network-config-analyzer/PeerContainer.py

```diff
@@ -105,7 +105,7 @@ def _set_namespace_list(self, ns_resources):
                 with open(path) as yaml_file:
                     code = yaml.load_all(yaml_file, Loader=yaml.SafeLoader)
                     for ns_code in code:
-                        if ns_code and ns_code.get('kind') == 'NamespaceList':
+                        if isinstance(ns_code, dict) and ns_code.get('kind') == 'NamespaceList':
```

Comment:
Shouldn't we fix as well at line 172: `if resource and resource.get('kind') == 'NamespaceList':` ?

Reply:
good catch. I found a few more